### DISCUSSION TITLE
支持tar包安装

### DISCUSSION THREAD

**Discussion by wzg-zhuo** [TuGraph-family/tugraph-db#158] [Ideas]
支持支持tar包安装，增加可移植性

**Comment by knightast** [comment]
现在能打包成zip包，解开配下环境就能跑。

移植性主要还是CPU架构是否兼容，OS的基础库是否一致，TuGraph基本都没问题，就是得在对应的环境编译一遍。

**Reply by wzg-zhuo** [reply]
是否可以发布不需要编译的tar包，直接解压使用

**Reply by knightast** [reply]
你可以这么理解，有这样的tar包，但一个tar包只能在某些操作系统使用。就像我们去网上下载软件，也需要指定是windows，还是ubuntu，或者centos

**Reply by wzg-zhuo** [reply]
所以是需要以源码编译形式来做

**Reply by knightast** [reply]
源码编译比较稳妥，也可以拿rpm或deb包试一把

**Reply by wzg-zhuo** [reply]
嗯呢，正在试~~，多谢 解答！
